elkodon/src/service/port_factory/event.rs

```diff
@@ -1,72 +1,117 @@
+//! # Examples
+//!
+//! ```
+//! use elkodon::prelude::*;
+//!
+//! # fn main() -> Result<(), Box<dyn std::error::Error>> {
+//! let event_name = ServiceName::new(b"MyEventName")?;
+//! let event = zero_copy::Service::new(&event_name)
+//!     .event()
+//!     .open_or_create()?;
+//!
+//! println!("name:                         {:?}", event.name());
+//! println!("uuid:                         {:?}", event.uuid());
+//! println!("max listeners:                {:?}", event.static_config().max_supported_listeners());
+//! println!("max notifiers:                {:?}", event.static_config().max_supported_notifiers());
+//! println!("number of active listeners:   {:?}", event.dynamic_config().number_of_listeners());
+//! println!("number of active notifiers:   {:?}", event.dynamic_config().number_of_notifiers());
```

Comment:
Not for now but for a rainy day. What do you think about implementing `Display` to get to this information?

Reply:
Added issue: #56